tag_database

```diff
@@ -264,6 +265,7 @@ sampleSize:array,random,intermediate
 scrollToTop:browser,intermediate
 sdbm:math,utility,intermediate
 serializeCookie:utility,string,intermediate
+serializeForm:browser,beginner,utility,string
```

Comment:
The expertise level of the snippet should be last - `browser,utility,string,beginner`